tests/samples/pkg_intree/backend/intree_backend.py

```diff
@@ -0,0 +1,2 @@
+def get_requires_for_build_sdist(config_settings):
+    return []
```

Comment:
Maybe return some dummy value just so the tests can check it's really this function being called.

Reply:
Done